### DISCUSSION TITLE
How to protect mailbook in production

### DISCUSSION THREAD

**Discussion by nezaboravi** [Xammie/mailbook#51] [Q&A] [answered]
Tried to set middleware, but somehow it skips it, and route is exposed on web. 
Is there a way to protect access to /mailbook?

**Comment by Xammie** [comment, marked as answer]
Similar to my answer on #50. This can be achieved with the following config in `config/mailbook.php`

```php
'middlewares' => [
    'web',
    'auth',
    Xammie\Mailbook\Http\Middlewares\RollbackDatabase::class,
],
```

The `auth` middleware would make sure only logged in users are able to see `/mailbook`.

You can also add another custom middleware if you only want admin users to access mailbook.

```php
'middlewares' => [
    'web',
    'auth',
    App\Http\Middlewares\AdminMiddleware::class, // middleware that would only allow users with the role of admin
    Xammie\Mailbook\Http\Middlewares\RollbackDatabase::class,
],
```

If this does not help, please give me some more context in the form of code or screenshots of what you are trying to do.

**Reply by nezaboravi** [reply]
Thank you, somehow missed #50
